<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuNew_Game">
     <property name="title">
      <string>New Game</string>
     </property>
     <addaction name="actionEasy"/>
     <addaction name="actionMedium"/>
     <addaction name="actionHard"/>
     <addaction name="actionInsane"/>
    </widget>
    <addaction name="menuNew_Game"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuDisplay_KeyPad">
     <property name="title">
      <string>Display KeyPad</string>
     </property>
     <addaction name="actionNo_Key_Pad"/>
     <addaction name="actionTop_Key_Pad"/>
     <addaction name="actionBottom_KeyPad"/>
     <addaction name="actionLeft_KeyPad"/>
     <addaction name="actionRight_KeyPad"/>
    </widget>
    <addaction name="menuDisplay_KeyPad"/>
   </widget>
   <widget class="QMenu" name="menu_Score">
    <property name="title">
     <string>&amp;Score</string>
    </property>
    <addaction name="actionShow_scores"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAutomatic_help"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_SudokuLink"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuWindow"/>
   <addaction name="menu_Score"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEasy">
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionInsane">
   <property name="text">
    <string>Insane</string>
   </property>
  </action>
  <action name="actionNo_Key_Pad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No KeyPad</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace, Ctrl+Del</string>
   </property>
  </action>
  <action name="actionTop_Key_Pad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top KeyPad </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionBottom_KeyPad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bottom KeyPad</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionLeft_KeyPad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left KeyPad</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionRight_KeyPad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Right KeyPad</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionShow_scores">
   <property name="text">
    <string>Show scores</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout_SudokuLink">
   <property name="text">
    <string>About SudokuLink</string>
   </property>
  </action>
  <action name="actionAutomatic_help">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
